### PR TITLE
feat: [C4R-775] added hover to app name and short description

### PR DESCRIPTION
**Pull request by OlteanuCosmin**
C4R-746 and C4R-785: selected tab remains in same state when user clicks on 'back' on all apps/users page
C4R-751: back button doesn't work properly on create app page
C4R-775: added hover to app name and short description (solves C4R-734 and C4R-708)
C4R-784: scroll bars are duplicated on windows on every page

### CODE REVIEW

File: frontend/src/index.css [hunk 1]
```diff
@@ -113,7 +113,7 @@
 
 .Application__Card__Description {
   display: -webkit-box;
-  -webkit-line-clamp: 4;
+  -webkit-line-clamp: 2;
```
Comment (dragos1195) [inline frontend/src/index.css:116]: Should be checked on multiple browsers.

File: frontend/src/pages/apps-store/AllApplications.tsx [hunk 2]
```diff
@@ -43,18 +43,18 @@ const AllApplications = () => {
...
+            className="flex pt-6 flex-col space-y-4 sm:space-y-0 sm:gap-x-4 sm:gap-y-4 flex-wrap lg:flex-row cursor-pointer select-none"
             aria-label="Tabs"
           >
             {APPLICATION_STORE_TABS.map((tab) => (
               <a
                 key={tab.name}
                 onClick={() => onTabClick(tab)}
                 className={classNames(
-                  selectedTab === tab.id
+                  tab.href === location.pathname.split('/')[2]
```
Comment (dragos1195) [inline frontend/src/pages/apps-store/AllApplications.tsx:54]: This should be in a UseEffect and we should use the selectedTab.
You should also check against other components with tabs.

File: frontend/src/pages/users/Users.tsx [hunk 3]
```diff
@@ -50,7 +50,7 @@ const Users = () => {
               key={tab.name}
               onClick={() => onTabClick(tab)}
               className={classNames(
-                selectedTab === tab.id
+                tab.href === location.pathname.split('/')[2]
```
Comment (dragos1195) [inline frontend/src/pages/users/Users.tsx:53]: Here also,

**Review by dragos1195 — CHANGES_REQUESTED** [review]

File: frontend/src/index.css (hunk 1 above)
Comment (OlteanuCosmin) [reply frontend/src/index.css:116]: Tested on Chrome, Firefox and Safari; works properly on all of them.

File: frontend/src/pages/apps-store/AllApplications.tsx (hunk 2 above)
Comment (OlteanuCosmin) [reply frontend/src/pages/apps-store/AllApplications.tsx]: Added additional useEffect.

**Pull request merged**